src/images/PreviewImage.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import { PictureOfDay } from "./DisplayPicture";
+import { fetchAPI, fetchRandomAPI } from "../utils/fetchData";
+
+export function PreviewImage() {
+
+    const [myPreviewImage, setMyPreviewImage] = useState<PictureOfDay | null>(null);
+    const [birthday, setBirthday] = useState('');
+
+    const handleRandomButtonClick = async () => {
```

Comment:
This doesn't seem to be running correctly but I don't think there's time to fix it so let's remove the random button from the page